packages/aragon-cli/src/commands/dao_cmds/acl_cmds/view.js

```diff
@@ -99,7 +99,7 @@ exports.handler = async function({
               }
             }
 
-            initAragonJS(dao, apm['ens-registry'], {
+            initAragonJS(dao, apm.ensRegistryAddress, {
```

Comment:
Is it `apm.ensRegistryAddress` or `apm.ensRegistry`? If it's not an error let's make it consistent for clarity or explain what each variable represents. 
`ensRegistryAddress` is also used in:
- https://github.com/aragon/aragon-cli/pull/895/files#diff-dbe6805e3cab1be07d9137504711cb05R59
- https://github.com/aragon/aragon-cli/pull/895/files#diff-f0a56a91e95462ee3c2a7f43e5ee4974R401
- https://github.com/aragon/aragon-cli/pull/895/files#diff-7fb765944652f71e14673b01c4e751d9R66